Platform/Teams/Tools and Improvements/[2019-06-11] State of E2E Testing on VA.gov.md

```diff
@@ -0,0 +1,53 @@
+# State of E2E Testing on VA.gov
```

Comment:
overall question: it seems like this is just about recording some notes about current state and noting some open questions --- what are you planning next in terms of following up to what's documented here?

Reply:
With as many of these points as I can take into consideration, I'm inclined to explore TestCafe as our primary E2E testing framework. It seems like a pretty good candidate for casting the widest net in terms of features we want.

1. It has a supported plugin for integration with Sauce Labs or BrowserStack for cross browser testing.
2. There's clear documentation for integrating with Jenkins for CI.
3. It's able to take screenshots like Puppeteer, so we can still do visual regression testing, although we will probably need to rework our current code to use TestCafe's APIs.
4. There is a package for accessibility. Although not directly under the same org as TestCafe, it seems to be well maintained and easy to use.
5. For creating or converting tests, there even seems to be an IDE that enables anyone, perhaps even non-devs, to create E2E tests without writing any code, which could ease the pain of transitioning to a new framework.

Reply:
First step might be to convert a test to TestCafe and try running it against Sauce labs.